README.md

```diff
@@ -136,6 +136,12 @@ Usage: `@Spec(path="firstName", spec=LikeIgnoreCase.class)`.
 
 There are also other variants which apply the wildcard only on the beginning or the ending of the provided value: `StartingWithIgnoreCase` and `EndingWithIgnoreCase`.
 
+There is an ability to set custom `Locale` in `@Spec.config`. If locale is not provided, then system default will be used.
+
+Usage: `@Spec(path="firstName", spec=LikeIgnoreCase.class, config="pl_PL")` 
```

Comment:
1. Please make it more concise, i.e. replace with just "Locale settings is important for case-insensitive searches. Please check the locale support section for details"   (link the 'locale-support section' with appropriate README section)
2. Please add similar comments to other IgnoreCase specs in the readme